middleware/logging.go

```diff
@@ -85,3 +86,28 @@ func (i *interceptor) Hijack() (net.Conn, *bufio.ReadWriter, error) {
 	}
 	return hj.Hijack()
 }
+
+// LogContext is a middleware to inject the organization and user id into
```

Comment:
FWIW, this doesn't really have anything to do with logging. `logging.With` is expected to take a general context and add whatever useful bits it can find in the context to the logs.

This is essentially https://github.com/weaveworks/common/blob/master/middleware/http_auth.go